booster/test/rpc-integration/test-collectiontest/response-from-12.json

```diff
@@ -93,7 +93,7 @@
                                                                                                 "name": "SortInt",
                                                                                                 "args": []
                                                                                             },
-                                                                                            "value": "6"
+                                                                                            "value": "4"
```

Comment:
The strange changes to this the output of `test-collectiontest` were caused by the reuse of `.json` output file by both `kore-rpc-booster` and `booster-dev` runs of the test. Since Booster's result is `stuck` (no more rules), `kore-rpc-booster` will fallback to `Kore` and the result will contain Kore's preferred set ordering.